core/iwasm/common/gc/gc_common.c

```diff
@@ -53,6 +77,26 @@ wasm_func_type_get_param_count(WASMFuncType *const func_type)
     return 0;
 }
 
+wasm_ref_type_t *
+wasm_ref_type_normalize(wasm_ref_type_t *ref_type)
+{
+    if (ref_type->value_type != REF_TYPE_HT_NULLABLE) {
+        return ref_type;
+    }
+    int32_t heap_type = ref_type->heap_type;
```

Comment:
use `int32` in our internal file, and put the variable declaration to the front of the function